### ISSUE TITLE
[SUBMIT] Build a blog with Vue and Webiny Headless CMS

### ISSUE THREAD

**Issue by Tammibriggs** [webiny/write-with-webiny#59]
### Introduction

If you have tried building a blog from scratch by writing the server-side code and then integrating it with the client side you can tell how much time and effort is required to complete it. But this isn't the case with the use of a CMS like Webiny where a ready-to-use Admin page for creating, editing, publishing, and managing content has been created for you, giving you as much time as needed to spend on the client side to create a gorgeous UI.

### Goals

In this tutorial, we will build a blog site with Vue.js and Webiny Headless CMS. We will be learning how to set up Webiny, create content models & data in Webiny CMS, and how to consume added data in a Vue.js application.

### Outline

- Introduction
- Prerequisites 
- Introduction to Webiny
- Setting up our Webiny Headless CMS
- Defining Content Model in the Admin App
- Adding content
- Building a blog with Webiny Headless CMS
  - Cloning the start Vue app
  - Setting up Roles and Permissions
  - Fetching all posts
  - Fetching single post
- Conclusion

**Comment by endymion1818** [comment]
Hi @Tammibriggs , similar to my comment on the other blog ... most of the others we have require some kind of user account setup. we're preparing some documentation on how to do that but it's not going to be ready for a little while unfortunately. If you can find another subject tht doesn't require user accounts, we'd happily consider it.

**Comment by Tammibriggs** [comment]
Ok. I understand.

**Comment by endymion1818** [comment]
On hold, let's get to this one after #61 is complete.

**Comment by Nziokitech** [comment]
hello @endymion1818 i would like to work on this, hope its open

**Comment by endymion1818** [comment]
Hey @Nziokitech, let's chat more in the community, please join and send me a DM https://www.webiny.com/slack/
